data/json/mapgen/standing_stones.json

```diff
@@ -0,0 +1,59 @@
+[
+  {
+    "name": "GROUP_STANDING_STONES",
+    "type": "monstergroup",
+    "default": "mon_null",
+    "monsters": [
+      { "monster": "mon_darkman", "freq": 100, "cost_multiplier": 1, "conditions" : ["NIGHT"] }
+    ]
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": [ "standing_stones" ],
+    "weight": 500,
+    "object": {
+      "rows": [
+        "ffffffffffffffffffff..ff",
+        ".ffffff....fffffffffff..",
+        "ff.ffffffff.....o...fff.",
+        "ff..f...ffffffff......ff",
+        ".fff.o.........f....ffff",
+        "f.....fff......fff....f.",
+        "f......f........ff......",
+        "ff....................o.",
+        ".f.o........o...........",
+        "...........ss...O.....ff",
+        ".........ossssss.......f",
+        ".......sssssssss.....fff",
+        "fff..OssssssOsss.o.....f",
+        "f......sssssssss.....o..",
+        "........ossssssO.......f",
+        "o.......ss..ss.........f",
+        "fff.......o.............",
+        "....................ffff",
+        ".ff....................f",
+        "f....o...............of.",
+        "fff..................f..",
+        "ffffffffff.....o...ffff.",
+        "ff.f.f.f.f....fff..f.f.f",
+        "ffffff..........ffffffff"
+      ],
+      "set": [ { "point": "trap", "id": "tr_brazier", "x": 10, "y": 12 } ],
+      "terrain": {
+        ".": [ "t_grass", "t_grass", "t_grass", "t_dirt", "t_dirt" ],
+        "f": [ "t_grass", "t_grass", "t_grass", "t_tree", "t_tree_young", "t_underbrush", "t_underbrush", "t_underbrush", "t_dirt" ],
+        "s": "t_dirt",
+        "o": "t_dirt",
+        "O": "t_dirt"
+      },
+      "furniture": {
+        "o": "f_boulder_small",
+        "O": "f_boulder_large"
+      },
+      "place_loot": [
+        { "item": "stick", "x": 10, "y": 12, "repeat": 2 }
```

Comment:
I'll submit a PR to your repo/branch showing how this can be mappable (just as furniture above).